### ISSUE TITLE
TypeError: Cannot read property 'destroy' of undefined (no mainMenuBar)

### ISSUE THREAD

**Issue by vitalets** [josdejong/jsoneditor#855]
When editor is used in textmode without `mainMenuBar`, it throws an error on destroy:
```
TypeError: Cannot read property 'destroy' of undefined
    at JSONEditor.textmode.destroy (jsoneditor.js:9759)
    at Editor.componentWillUnmount (Editor.jsx:37)
    at callComponentWillUnmountWithTimer (react-dom.development.js:21896)
    at HTMLUnknownElement.callCallback (react-dom.development.js:336)
```

This is because instance of `FocusTracker` is [created only when `mainMenuBar` is used](https://github.com/josdejong/jsoneditor/blob/8894420263e2e63de340227f481c304a377ac168/src/js/textmode.js#L155).
But on destroy there is [no check.](https://github.com/josdejong/jsoneditor/blob/8894420263e2e63de340227f481c304a377ac168/src/js/textmode.js#L614)

**Comment by josdejong** [comment]
Thanks Vitaliy for reporting, this issue was introduced a few days ago in `v7.3.0`. This `FocusTracker` should be always created, not only when `mainMenuBar` is used. Fixed via 1026d9102c2567a34136a10770c6eafd16512a1c.

**Comment by josdejong** [comment]
Fixed now in `v7.3.1`.

**Comment by vitalets** [comment]
Thank you for quick fix! 👍
